docs/ecctl-getting-started.asciidoc

```diff
@@ -252,47 +252,292 @@ https://golang.org/pkg/text/template/[`text/templates`] on demand.
+            "version": "7.6.0" <12>
+          }
+        }
+      }
+    ]
+  }
+}
+--
+<1> The region for the Elasticsearch cluster
+<2> Instance configuration ID
+<3> The number of availability zones for the Elasticsearch cluster
+<4> The amount of memory allocated for each Elasticsearch node
+<5> The version of the Elasticsearch cluster
+<6> The template on which to base the deployment
+<7> The number of availability zones for Kibana
+<8> The amount of memory allocated for Kibana
+<9> The version of the Kibana instance
+<10> The number of availability zones for APM
+<11> The amount of memory allocated for APM
+<12> The version of the APM instance
+
+This JSON contains the settings for a highly available Elasticsearch cluster deployed across two availability zones, a single instance of Kibana, and a single APM server.
+
+Run the <<ecctl_deployment_create,{p} deployment create>> command with `create-deployment.json` as a parameter. For this and other commands, you can add an optional `--track` parameter to monitor the progress.
+
+[source, sh]
+--
+ecctl deployment create -f create-deployment.json
+--
+
+[source, sh]
 --
-ecctl deployment create --file ./deployment.json --name=my-new-deployment --version=7.4.1
 {
   "created": true,
-  "id": "c44ad3fa558a237d0c17d0f4273801df",
-  "name": "my example cluster",
+  "id": "7229888e7bf8350c7e4d07d7374171c0",
+  "name": "My first ecctl deployment",
   "resources": [
     {
-      "cloud_id": "my_elasticsearch_cluster:REDACTED",
+      "cloud_id": "My_first_ecctl_deployment:dXMtY2VudHJhbDEuZ2NwLmZvdW5kaXQubm8kYjFlZWVjOGQ0YWVlNGY3ZDgxNTM2Zjc1ZjZhN2Y1MDgkM2ViZTAzNmI0NDhkNDc3Y2E2ZTJjZTQ5NmE4ZDQ5ODA=",
       "credentials": {
-        "password": "REDACTED",
+        "password": "qIJuMbL8BFfgoNq5bpjQhDRC",
         "username": "elastic"
       },
-      "id": "edd8b0bf7ece46dab47acc7074a7f191",
+      "id": "b1eeec8d4aee4f7d81536f75f6a7f508",
+      "kind": "elasticsearch",
+      "ref_id": "main-elasticsearch",
+      "region": "gcp-us-central1"
+    },
+    {
+      "elasticsearch_cluster_ref_id": "main-elasticsearch",
+      "id": "3ebe036b448d477ca6e2ce496a8d4980",
+      "kind": "kibana",
+      "ref_id": "main-kibana",
+      "region": "gcp-us-central1"
+    },
+    {
+      "elasticsearch_cluster_ref_id": "main-elasticsearch",
+      "id": "5a03472f6dfe4f17acbe62622823b9cb",
+      "kind": "apm",
+      "ref_id": "main-apm",
+      "region": "gcp-us-central1",
+      "secret_token": "zfufcfe15eCVJk78b5"
+    }
+  ]
+}
+--
+
+The response indicates that the request was submitted successfully. It includes the `elastic` user password, which you can use to log in to Kibana or to access the Elasticsearch REST API. Make a note of the deployment ID, which you will use in the next example.
+
+[id="{p}-example-update-deployment"]
+=== Update a deployment
+
+Now that you have used {p} to create a deployment, you can scale it up, by increasing the size of the Elasticsearch data nodes from 4096 to 8192 MB.
```

Comment:
The previous JSON payload (create-deployment.json) has size set to 1024 and this one updating to 4096 :)
```suggestion
Now that you have used {p} to create a deployment, you can scale it up, by increasing the size of the Elasticsearch data nodes from 1024 to 4096 MB.
```